mods/cnc/rules/world.yaml

```diff
@@ -265,6 +265,7 @@ World:
 EditorWorld:
 	Inherits: ^BaseWorld
 	EditorActorLayer:
+	EditorCursorLayer:
```

Comment:
I guess this needs an update rule.

Reply:
Update rules aren’t able to do these kind of changes. It will need to be added to https://github.com/OpenRA/OpenRAModSDK/wiki/Update-notes:-hotfix-release-to-next-release instead.